config/runtime.exs

```diff
@@ -216,6 +216,9 @@ cond do
       username: get_env("SMTP_USERNAME"),
       password: get_env("SMTP_PASSWORD")
 
+  get_env("TZDATA_AUTOUPDATE_DISABLED") ->
```

Comment:
This should not be set in the conditions linked to the mailer. You can set it like the line 179 (with an `if`)